docs/Concepts/Data-Storage-Formats.md

```diff
@@ -0,0 +1,64 @@
+---
+description: Data storage formats
+---
+
+# Data storage formats
+
+Besu offers two formats for storing the world state, [Forest of Tries](#forest-of-tries) and [Bonsai Tries](#bonsai-tries).
+You can [configure the data storage format](../HowTo/Configure/Configure-Data-Storage.md) when starting Besu.
+
+## Forest of Tries
+
+Forest of Tries, also called forest mode, is the default storage format.
+
+In forest mode, each node in the trie is saved in a key-value store by hash. For each block, the world state is updated
+with new nodes, leaf nodes, and a new state root. Old leaf nodes remain in the underlying data store. Data is accessed
+and stored by hash, which increases the size of the database and increases the resources and time needed to access account data.
+
+![forest_of_tries](../images/forest_of_tries.png)
```

Comment:
These diagrams look great! To reiterate my suggestions:

- Use the gray background color, like in the other Besu diagrams [here](https://www.figma.com/file/5JXmpDvfsOHmDkCG6biRTJ/Pegasys-Diagrams?node-id=1%3A2).
- Arrows:
    - Make sure they don't obscure each other when overlapping.
    - Make sure the pixels line up at the beginning of the arrow, and leave a little space between the arrow head and target.
    - Try to avoid using diagonal lines.
- Use sentence-style capitalization for labels.
- I would also round the dark box edges by 2 pixels.

Reply:
Looks much better - can you add the diagram to [this Figma page](https://www.figma.com/file/5JXmpDvfsOHmDkCG6biRTJ/Pegasys-Diagrams?node-id=1%3A2)?